src/pudl/transform/nrelatb.py

```diff
@@ -0,0 +1,417 @@
+            "heat_rate_mmbtu_per_mwh",
+            "capex_construction_finance_factor",
+            "capex_grid_connection_per_kw",
+            "capex_overnight_per_kw",
+            "capex_overnight_additional_per_kw",
+            "heat_rate_penalty",
+            "net_output_penalty",
+        ],
+    )
+
+    @property
+    def core_metric_parameters_all(self: Self) -> list[str]:
+        """Compilation of all of the parameter values from each of the tables.
+
+        Also check if there are no duplicate core_metric_parameters. We expect all of the parameters
+        across the :class:`TableUnstacker` to be unique.
+        """
+        core_metric_parameters_all = (
+            self.rate_table.core_metric_parameters
+            + self.scenario_table.core_metric_parameters
+            + self.tech_detail_table.core_metric_parameters
+        )
+        assert not {
+            x
+            for x in core_metric_parameters_all
+            if core_metric_parameters_all.count(x) > 1
+        }
+        return core_metric_parameters_all
+
+
+@asset
+def _core_nrelatb__transform_start(raw_nrelatb__data):
+    """Transform raw NREL ATB data into semi-clean but still very skinny table."""
+    ## EARLY CLEANING/DEDUPING
+    # rename dict f semi-cleaned param name -> desired pudl name
+    # all the core_metric_parameters are here even if we aren't renaming mostly
+    # to make sure we've got em all.
+    core_metric_parameters_rename = {
+        "gcc": "capex_grid_connection_per_kw",
+        "interest_rate_nominal": "interest_rate_nominal",
+        "debt_fraction": "debt_fraction",
+        "inflation_rate": "inflation_rate",
+        "calculated_interest_rate_real": "interest_rate_calculated_real",
+        "fixed_o&m": "opex_fixed_per_kw",
+        "fcr": "fixed_charge_rate",
+        "crf": "capital_recovery_factor",
+        "tax_rate_(federal_and_state)": "tax_rate_federal_state",
+        "calculated_rate_of_return_on_equity_real": "rate_of_return_on_equity_calculated_real",
+        "heat_rate": "heat_rate_mmbtu_per_mwh",
+        "net_output_penalty": "net_output_penalty",
+        "variable_o&m": "opex_variable_per_mwh",
+        "interest_during_construction_-_nominal": "interest_rate_during_construction_nominal",
+        "cf": "capacity_factor",
+        "capex": "capex_per_kw",
+        "fuel": "fuel_cost",
+        "rate_of_return_on_equity_nominal": "rate_of_return_on_equity_nominal",
+        "occ": "capex_overnight_per_kw",
+        # this only applies to technology_description's Coal_Retrofits & NaturalGas_Retrofits (maybe we should combine)
+        "additional_occ": "capex_overnight_additional_per_kw",
+        "wacc_nominal": "wacc_nominal",
+        "wacc_real": "wacc_real",
+        "cfc": "capex_construction_finance_factor",
+        "lcoe": "levelized_cost_of_energy_per_mwh",
+        "heat_rate_penalty": "heat_rate_penalty",
+    }
+    nrelatb = (
+        raw_nrelatb__data.replace(["*", ""], pd.NA)
```

Comment:
You mentioned that `*` is "wildcard", which is probably different from "no value at all." My understanding of why it's OK to merge these into the same `pd.NA` is:

* `*` appears in index columns when they mean "wildcard" but doesn't appear in data columns
* `""` appears in data columns when they mean "no value" but doesn't appear in index columns

Is that right?

Reply:
yes this is exactly right. but i appreciate you for commenting about this bc it made me go see if I could remove the * from this null-ification. because conceptually what is happening with mostly of the *'s in the index columns is that those columns are being dropped for the records that have any *'s in them during the unstacking (i.e. when there is a *'s in the index it **almost** always means that record should be a part of a table which has an idx w/o the column containing *'s). this is true for almost all of the cases except for the some of the `cost_recovery_period_years` `core_nrelatb__yearly_financial_cases_by_scenario` (which rn has some nulls in it so i can't add this column to the pks of the table).

imma go see if i miscategorized a parameter or if i should broadcast these records across the existing `cost_recovery_period_years`.

Reply:
okay i did not miscategorize a param (which is to say the values are non-unique in the for these params in the fin cases tbl w/o the `cost_recovery_period_years` in the idk).

so i did a quick broadcasting of the `*` (mirroring what i did for the fcr) and it worked well but it resulted in records which still have a nulls in that column. because some of the records didn't match the idx values (they were `left_only` records in the broadcast merge). i'm sure i could find a defensible way to broadcast values into that column but i am going to lay this down for now for time sake.